ui/raidboss/timeline.js

```diff
@@ -582,8 +582,8 @@ class TimelineUI {
     this.init = true;
 
     this.root = document.getElementById('timeline-container');
-    if (Options.Language)
-      this.root.classList.add('lang-' + Options.Language);
+    if (Options.TimelineLanguage)
```

Comment:
Hmm.  This isn't quite true either, because `TimelineLanguage` could be unset.

Maybe it'd be worth doing something like:
`this.options.TimelineLanguage = this.options.TimelineLanguage || this.options.ParserLanguage || 'en';`

(or storing that in some other `this.timelineLanguage` variable if you don't want to modify options?)

And then TimelineLanguage will be set correctly.  Without this, if you don't set timeline language explicitly, then you won't get the CSS.  (Previously you only got the CSS for the parser language, which was also wrong.)

Reply:
remove `if (Options.TimelineLanguage)` and adding 
```javascript
this.root.classList.add('lang-' + Options.TimelineLanguage || Options.ParserLanguage || 'en');
```
would fix this.